dll/win32/shlwapi/shlwapi.spec

```diff
@@ -551,19 +551,19 @@
 551 stub -noname IShellFolder_CompareIDs
 552 stdcall -stub -noname -version=0x501-0x502 SHEvaluateSystemCommandTemplate(wstr ptr ptr ptr)
 553 stdcall IsInternetESCEnabled()
-554 stdcall -noname -stub SHGetAllAccessSA()
+554 stdcall -stub -noname -version=0x502 SHGetAllAccessSA()
```

Comment:
Not sure where you're getting your information from but my Win2003 version of shlwapi has `_SHGetAllAccessSA@0` as ordinal 554 according to symbols. I will assume the same is true for all the other functions unless you have proof otherwise.

Reply:
Oh. I just realized you're marking them as 5.2.
Then this seems merely pointless, not wrong.

Reply:
> pointless

There are already some cases:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=version%3D0x50

Would that not help with "AppCompat" someday? At least as documentation. Especially such functions/ordinals which differ between Windows versions.